docker/Dockerfile

```diff
@@ -211,6 +211,12 @@ RUN mkdir .duckdb/ && chmod -R a+rwX .duckdb/ && \
 RUN sed -i "s/^#shared_preload_libraries = ''/shared_preload_libraries = 'pg_search,pg_analytics,pg_cron'/" /usr/share/postgresql/postgresql.conf.sample && \
     echo "cron.database_name = 'postgres'" >> /usr/share/postgresql/postgresql.conf.sample
 
+# Barman is required for backups in CloudNativePG
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    barman \
+    barman-cli-cloud \
+    && rm -rf /var/lib/apt/lists/*
+
```

Comment:
Apart from Barman itself you need the S3, Azure and GCS adapters, which technically have their own Debian packages, but they are a lot bigger than just the bits that Barman needs. For that reason installation via `npm` is recommended. The addition to the image will be overall smaller.

Reply:
My understanding from this link is that `barman-cli-cloud` encompasses all of them already: https://docs.pgbarman.org/release/3.8.0/#barman-client-utilities-for-the-cloud-barman-cli-cloud

We previously add it working on AWS with just these two installations.

Reply:
Oh, lol! Thank you. Whichever approach you think is best then :)

Reply:
I would prefer we stick to Debian than `pip` though, how much bigger are the packages?

Reply:
I'll take a look again on Monday and post the exact numbers.

Reply:
I will try to get you an APT only solution if it is feasible.

Reply:
Appreciate it, if Python really is better than so be bit, but I have my doubts haha. Apt is a lot easier to work with